js/core/utils/size.js

```diff
@@ -27,49 +27,106 @@ export const getElementBoxParams = function(name, elementStyles) {
     };
 };
 
-const getBoxSizingOffset = function(name, elementStyles, boxParams) {
-    const size = elementStyles[name];
-
-    if(elementStyles.boxSizing === 'border-box' && size.length && size[size.length - 1] !== '%') {
-        return boxParams.border + boxParams.padding;
-    }
-
-    return 0;
-};
 const getElementComputedStyle = function(element) {
     const view = element?.ownerDocument?.defaultView || window;
     return view.getComputedStyle && view.getComputedStyle(element);
 };
+const getCSSProperty = function(element, styles, name, defaultValue) {
+    let result = styles && styles[name];
+    if(result === '' && element.style) {
+        result = element.style[name];
+    }
+    if(result === '' || result === undefined) {
+        return defaultValue;
+    }
+    return result;
```

Comment:
We can simlify method in case the style value is always string.
```suggestion
    return styles?.[name] || element.style?.[name] || defaultValue
```